explainer.md

```diff
@@ -88,6 +89,21 @@ Items that are designed to be purchased once and last permanently in the user’
 itemService.acknowledge(purchaseToken, 'onetime');
 ```
 
+### Checking existing purchases
+
+The `listPurchases` method allows a client to get a list of items that are currently owned or purchased by the user. This may be necessary to check for entitlements (eg. whether a subscription, promotional code, or premium upgrade is active) or to recover from flaky network issues during a purchase (eg. item is purchased but not yet acknowledged).
```

Comment:
Perhaps premium -> onetime?

Reply:
These are meant to be examples of in-app purchases that might remain owned, not references to terms in the doc. Changed "premium" to "temporary" to try to differentiate.